src/options/options.requests.ts

```diff
@@ -49,6 +49,7 @@ export interface CalculateGammaResponse {
 
 export interface GreekQueryRequest extends PanopticRequest {
   address: string;
+  panopticPool: string,
```

Comment:
Is PanopticPool required for every request? I think probably not - for example, some of the Greek queries might only need to know a pair of tokens, or some of the CollateralToken-only requests like withdraw might not need it.

Assuming its sometimes needed, sometimes not: Is there an approach where we can make panopticPool an optional parameter of PanopticRequest, so it doesn't have to be repeated for every request in this file?

Or, if it actually is necessary for each request, maybe we make it a mandatory field of PanopticRequest?

Reply:
Done: 2999f679ade7548186380114018680bbe65cb846

Reply:
Done: 7aff22d5fb8ddba6dd413e21aea46f14175080bf